src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -392,6 +400,26 @@ public void sendRegisterCustomSettingsRequest(TransportService transportService)
         }
     }
 
+    /**
+     * Requests that OpenSearch register all the Interfaces implimented by the extension.
+     *
+     * @param transportService  The TransportService defining the connection to OpenSearch.
+     */
+    public void sendRegisterImplimentedInterfacesRequest(TransportService transportService) {
```

Comment:
I made a comment over on the OpenSearch PR that I don't think we need to do a separate request for this.  During extension initialization, OpenSearch sends an `ExtensionInitRequest` and I think including the values in the response to that is a better implementation point.

Reply:
That is the first request sent by opensearch once it gets to know what all extensions are registered I believe. Cool i will try to add this along with whatever sdk is sending for this request.

Reply:
I have added the response with ExtensionInitResponse